deployment/pypi/README.md

```diff
@@ -47,15 +47,14 @@ This is the PyPI build and upload tool for NNI project.
   powershell
   Python >= 3.5
   Pip
-  Node.js
   Yarn
-  tar
   ```
 
 * __How to build__
-
+ 
+  parameter `version_os` is used to build for Windows 64-bit or 32-bit. 
   ```bash
-  powershell ./install.ps1
+  powershell ./install.ps1 -version_os [64/32]
```

Comment:
The windows version can be detected automatically, it's better to handle the logic ourselves, instead of asking users.

Reply:
Hi Chi, this is a merge back PR, could you  please open an issue, so we can review and fix it later?   Normally we do not change things in merge back PR.